plugins/foreman_ansible/0.2/index.md

```diff
@@ -0,0 +1,96 @@
+---
+layout: plugin
+title: Foreman Ansible 0.2 Manual
+version: 0.2
+---
+
+# 1. Introduction
+
+Foreman Ansible allows you to import hosts via Ansible, along with facts about these hosts and reports of their playbook runs. This plugin should be compatible with any version of Foreman above 1.8, including 1.8.
+
+Every time you run a playbook or an Ansible module, Foreman will receive facts and a report for the host you have executed it on.
+
+![host with reports](static/images/plugins/foreman_ansible/registered_host.png)
+
+# 2. Installation
+
+## 2.1 Ansible callback
+
+In order to make Ansible send us data from the hosts, we set up a callback on your host that runs Ansible. To do this:
+
+* Change your /etc/ansible/ansible.cfg to contain the following options
+
+      callback_plugins = ~/.ansible/plugins/callback_plugins/
+      bin_ansible_callbacks = True
+
+* And run this line to download the callback file in the appropriate location
+
+      wget https://raw.githubusercontent.com/theforeman/foreman_ansible/master/extras/foreman_callback.py -O ~/.ansible/plugins/callback_plugins/foreman_callback.py
+
+* Now open the script to set up `FOREMAN_URL` and `FOREMAN_SSL_CERT` variables on the callback. Alternatively, you can find the source for the [Ansible callback here](https://github.com/theforeman/foreman_ansible) and deploy it in your directory of choice.
+
+## 2.2 Plugin
+### 2.2.1 Installer (recommended on Foreman 1.11+)
```

Comment:
On the proxy side - nothing - there's no proxy component for the moment, at least as far as foreman_ansible is concerned. Remote execution is a whole another thing that is not merged yet. But I'll remove the installer section until 1.11 then!